pkgs/development/libraries/qt-6/examples/qtbase/i18n.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, qtbase
+, qmake
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qtbase-example-i18n";
+  inherit (qtbase) version src;
```

Comment:
What about `src = "${qtbase.src}/examples/widgets/tools/i18n";`

Reply:
this only works with `src = fetchgit`

qtbase.src is a tar.xz file

```
unpacking sources
unpacking source archive /nix/store/xdlkvzjshjh8q5ym69hb2c9x2qaargqn-qtbase-everywhere-src-6.4.2.tar.xz/examples/widgets/tools/i18n
tar: /nix/store/xdlkvzjshjh8q5ym69hb2c9x2qaargqn-qtbase-everywhere-src-6.4.2.tar.xz/examples/widgets/tools/i18n: Cannot open: Not a directory
tar: Error is not recoverable: exiting now
do not know how to unpack source archive /nix/store/xdlkvzjshjh8q5ym69hb2c9x2qaargqn-qtbase-everywhere-src-6.4.2.tar.xz/examples/widgets/tools/i18n
```